compiler/src/iree/compiler/Pipelines/Pipelines.cpp

```diff
@@ -31,6 +33,18 @@
 namespace mlir {
 namespace iree_compiler {
 
+#if IREE_TRACING_FEATURES & IREE_TRACING_FEATURE_INSTRUMENTATION
+#define IREE_TRACE_BEGIN_PHASE(name)                             \
+  passManager.addPass(IREE::Util::createTraceFrameMarkBeginPass( \
```

Comment:
when the pass takes a StringRef (or const char*) the macro will just pass that along - then it's just like IREE_TRACE_FRAME_MARK_BEGIN_NAMED itself (which takes the literal passed to it directly)